frontend/jupyter/cypress/fixtures/pvcs.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pvcs":[],
+  "success":true,
+  "status":200,
+  "user": null
+}
```

Comment:
Add new line at end of file

Reply:
Why exactly? I'm not doubting you, I just want to understand.

Reply:
It's a convention? It's also the only file github is complaining about for it.

Reply:
Ok thanks, you are correct that github is complaining for this file and a couple of the other new ones. I'll go do that.